data_safe_haven/config/config.py

```diff
@@ -168,20 +167,13 @@ class ConfigSectionSRE(BaseModel, validate_assignment=True):
     @field_validator("databases")
     @classmethod
     def all_databases_must_be_unique(
-        cls, v: list[DatabaseSystem]
+        cls, v: list[DatabaseSystem | str]
     ) -> list[DatabaseSystem]:
-        if len(v) != len(set(v)):
+        v_ = [DatabaseSystem(d) for d in v]
+        if len(v_) != len(set(v_)):
             msg = "all databases must be unique"
             raise ValueError(msg)
-        return v
-
-    @field_serializer("software_packages")
-    def software_packages_serializer(
-        self,
-        packages: SoftwarePackageCategory,
-        info: FieldSerializationInfo,  # noqa: ARG002
-    ) -> str:
-        return packages.value
```

Comment:
Is this not needed?

Reply:
I don't think it is. When using `model_dump(mode="json")` I think Enums will be implicitly converted (it's working for me at least).

Reply:
That's great, saves some unnecessary code.